Content.Server/Botany/SeedPrototype.cs

```diff
@@ -176,7 +176,7 @@ public partial class SeedData
     [ViewVariables(VVAccess.ReadWrite)]
     [DataField("harvestRepeat")] public HarvestType HarvestRepeat = HarvestType.NoRepeat;
 
-    [DataField("potency")] public float Potency = 1f;
+    [DataField("potency")] public float BasePotency = 1f;
```

Comment:
Please don't rename this. Rather, update the comment documenting what this variable now does and how it's different from before.